WORKSPACE

```diff
@@ -6,3 +8,19 @@ emsdk_repo()
 
 # Install the Kotlin-Native compiler
 kotlin_native_repo()
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+rules_kotlin_version = "legacy-modded-0_26_1-02"
+rules_kotlin_sha = "245d0bc1511048aaf82afd0fa8a83e8c3b5afdff0ae4fbcae25e03bb2c6f1a1a"
+http_archive(
+    name = "io_bazel_rules_kotlin",
+    urls = ["https://github.com/cgruber/rules_kotlin/archive/%s.zip" % rules_kotlin_version],
```

Comment:
Should we try to unify the kotlin-native vs kotlin dependency?

Reply:
We could fork the official repo like cgruber did, but I think we need to think through the toolchain / auto_wasm stuff before we go there.